recipe/meta.yaml

```diff
@@ -1,14 +1,14 @@
 {% set name = "semantikon" %}
-{% set version = "0.0.19" %}
-{% set python_min = "3.10" %}
+{% set version = "0.0.20" %}
+{% set python_min = "3.11" %}
```

Comment:
I do not think this is necessary as you still support the numpy versions below 2.x.